playback/src/decoder/passthrough_decoder.rs

```diff
@@ -35,16 +34,14 @@ pub struct PassthroughDecoder<R: Read + Seek> {
     setup: Box<[u8]>,
 }
 
-pub struct PassthroughError(ogg::OggReadError);
-
 impl<R: Read + Seek> PassthroughDecoder<R> {
     /// Constructs a new Decoder from a given implementation of `Read + Seek`.
-    pub fn new(rdr: R) -> Result<Self, PassthroughError> {
+    pub fn new(rdr: R) -> DecoderResult<Self> {
         let mut rdr = PacketReader::new(rdr);
-        let stream_serial = SystemTime::now()
+        let since_epoch = SystemTime::now()
             .duration_since(UNIX_EPOCH)
-            .unwrap()
-            .as_millis() as u32;
+            .map_err(|e| AudioError::PassthroughDecoder(e.to_string()))?;
```

Comment:
Under what circumstances can this fail? If it never should, then should we indeed handle that gracefully or would a panic actually be in order?

Reply:
It fails if it is 1969.

Reply:
Is there anything to do with this that needs changes or can I Resolve this?

Reply:
I'm OK with it.